sbin/install.config.in.in

```diff
@@ -0,0 +1,89 @@
+VERSION=@PACKAGE_VERSION@
```

Comment:
good! I, my-self, would add a common prefix to all, such as CONFIG_, but not extremely important.